lib/provider/home_screen_provider.dart

```diff
@@ -79,7 +79,7 @@ class HomeScreenProvider extends ChangeNotifier {
 
   void sortPriceAscending() {
     productList.sort((a, b) =>
```

Comment:
Refrain from using single lettered variable names. Name it so its easy to understand what the variables contain